src/include/rear/ui.rb

```diff
@@ -68,6 +68,23 @@ def UsedModules
       deep_copy(modules)
     end
 
+    # returns a list of mountpoints that have to be included in BACKUP_PROG_INCLUDE
+    def GetMountpoints
+      # -D excludes pseudo filesystems
+      # -r raw output, no ascii-art
+      # -n no column header
+      cmd = "findmnt -D -t notmpfs,nodevtmpfs -o TARGET -n -r | sort"
```

Comment:
and btw findmnt have --json output which can be more useful

and -t looks like -O instead, not? `-O, --options <list>   limit the set of filesystems by mount options`